src/device_trezor/device_trezor.cpp

```diff
@@ -267,6 +495,10 @@ namespace trezor {
         cpend.key_images = key_images;
 
         // KI sync
+        for(size_t cidx=0, trans_max=unsigned_tx.transfers.second.size(); cidx < trans_max; ++cidx){
+          signed_tx.key_images[cidx] = unsigned_tx.transfers.second[cidx].m_key_image;
```

Comment:
Is signed_tx.key_images guaranteed to have enough size at that place ?

Reply:
Yes, it is set before for-loop
```cpp
signed_tx.key_images.resize(unsigned_tx.transfers.second.size());
```